pallets/elections/src/lib.rs

```diff
@@ -20,7 +20,7 @@ pub mod pallet {
     use frame_election_provider_support::{
         ElectionDataProvider, ElectionProvider, Support, Supports,
     };
-    use frame_support::pallet_prelude::*;
+    use frame_support::{pallet_prelude::*, traits::Get};
```

Comment:
do you need this import?

Reply:
Get? Yes, it's needed for `SessionPerdiod`.